Add metadata for research software support workshop (#120)
esciencecenter-digital-skills/workshop-metadata: ds-research-software-support/description.md

~150 words for on Eventbrite registration page
## Example

Research software is a topic of growing interest for many researchers. This workshop is tailored for support staff who are already familiar with RDM and the FAIR principles but would like to learn more about how to manage software better. 

The workshop will cover the following key aspects:
- How is software being developed? What is research software and how much do I need to know?
- What are Software Management Plans (SMPs) and how can researchers benefit from them? How can I help researchers to fill in SMPs?
- What is FAIR software, and what do I need to know to support researchers?

> - hat software licenses exist and how do I help researchers to pick the right one?

Reviewer: ```suggestion
- What software licenses exist and how do I help researchers pick the right one?
```
